lib/configuration.rb

```diff
@@ -55,11 +51,12 @@ def load_yaml(name, this_location, this_ref)
   end
 
   def symbolize(obj)
-    if obj.is_a? Hash
+    case obj
+    when Hash
```

Comment:
Several spots converted from IF to CASE. This one is interesting because the `===` operator can be type checking, so this is especially clean.